src/applications/hca/tests/e2e/hca-aiq.cypress.spec.js

```diff
@@ -47,9 +47,7 @@ describe('HCA-AIQ', () => {
   it('works with AIQ (Yes selected) - C12901', () => {
     cy.visit(manifest.rootUrl);
     cy.wait(['@mockUser', '@mockFeatures', '@mockEnrollmentStatus']);
-    cy.findAllByText(/apply.+health care/i, { selector: 'h1' })
-      .first()
-      .should('exist');
+    cy.findByTestId('form-title').contains(/apply.+health care/i);
```

Comment:
Is there a way to find these elements without adding a test ID? This isn't a pattern we use anywhere else in our tests and it's generally better to not add things to our code just for testing purposes.

Reply:
It's a E2E best practice as [documented by Platform QA Standards team](https://depo-platform-documentation.scrollhelp.site/developer-docs/Cypress-Best-Practices-on-VA.gov-Platform.1870430211.html#CypressBestPracticesonVA.govPlatform-data-testidAttribute).  Cypress.io also has a [more general best-practice](https://docs.cypress.io/guides/references/best-practices#Selecting-Elements) incl. not selecting elements by text-content.  data-testid's just about the only way to select elements in a non-brittle way.  Hope you'll agree!  :)

Reply:
I apologize, I did not realize it was documented as a best practice. Thank you for pointing me to the documentation.

Reply:
Thx much, Katie!  Resolving this convo.